File: Backend/tasks3Rn/Tasks3Rn/Model/TaskModel.cs
using System.ComponentModel.DataAnnotations;

namespace Tasks3Rn.Models
{
    public class TaskModel
    {
        public int Id { get; set; }

        [Required]
        [StringLength(100)]
        public string NomeTask { get; set; }
        public string DescricaoTask {  get; set; }
        public bool isTaskConcluida { get; set; }
    }
}

File: Frontend/CrudAngular/src/app/components/tasks/Models/TaskModel.ts
export class TaskModel {
  id: number; // O id pode ser opcional
  nomeTask: string;
  descricaoTask: string;
  isTaskConcluida: boolean;

  constructor(
    id: number,
    nomeTask: string,
    descricaoTask: string,
    isTaskConcluida: boolean,
  ) {
    this.id = id;
    this.nomeTask = nomeTask;
    this.descricaoTask = descricaoTask;
    this.isTaskConcluida = isTaskConcluida;
  }
}
